### PR TITLE
Add progress bar for translations

### PR DESCRIPTION
**Pull request by markedmondson**
Adds a progress bar for translations through API

Screenshot 2024-11-07 at 9 44 13 PM

### CODE REVIEW

**Review by glebm — APPROVED** [review]

**Comment by glebm** [comment]
Thanks! Please address the lint errors and I'll merge :+1:

File: lib/i18n/tasks/translators/yandex_translator.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@ def initialize(*)
 
     def translate_values(list, **options)
       list.map { |item| translator.translate(item, options) }
```
Comment (glebm) [inline lib/i18n/tasks/translators/yandex_translator.rb:19]: This method should return the result

Comment (markedmondson) [reply lib/i18n/tasks/translators/yandex_translator.rb:19]: Ah yeah good spot.

These translators are tricky without tests (and much exposure to the code before now)

Comment (markedmondson) [reply lib/i18n/tasks/translators/yandex_translator.rb]: [0ecba9d](https://github.com/glebm/i18n-tasks/pull/606/commits/0ecba9da23fa7c50506c2f3020d257ee14b7ed8a)

**Pull request merged**